public/views.py

```diff
@@ -21,14 +21,15 @@ def get_context_data(self, **kwargs):
         published_posts = public_posts.filter(pub_date__lte=timezone.now())
         start_today = datetime.datetime.combine(datetime.datetime.today(), datetime.time(0, 0, 0))
         training = Categ.objects.get(name="training")
-        yes = PartChoice.objects.filter(choicetext="yes")[0]
+        yes = PartChoice.objects.filter(choice="y")[0]
         next_trainings = (
             Event.objects.filter(categ=training).order_by("-date").filter(date__gte=timezone.make_aware(start_today))
         )
         if len(next_trainings) > 0:
             next_training = next_trainings[0]
             participants = next_training.participation_set.all()
-            partn = sum([1 for p in participants if p.part == yes])
+            coming = [p for p in participants if p.part == yes]
+            partn = sum([1 for p in coming])
```

Comment:
```suggestion
            partn = len(coming)
```

Reply:
sehr gut, ja in dem view sind noch einige veraltete Bauteile, die ich aus bequemlichkeit nicht rausgenommen habe, aber ich mache das mal noch in dem PR